themes/SuiteP/css/editview.scss

```diff
@@ -1191,6 +1191,8 @@ p.error {
 
 .error {
   border: none;
+  color: #f08377;
+  background: transparent;
```

Comment:
This will change all error messages. Consider creating a specialisation class for this purpose?